webhooks-extension/config/release/gcr-tekton-webhooks-extension.yaml

```diff
@@ -161,15 +161,22 @@ spec:
       - name: pull-request
         type: pullRequest
     params:
-      - name: pipelinerun
-        description: The name of the pipelinerun to be monitored
+      - name: pipelineruns
+        description: The name of the pipelineruns (and their namespace) to be monitored in format pipelinerun1:namespace1, pipelinerun2:namespace2
         default: default pipelinerun
       - name: commentsuccess
         description: The text of the success comment
-        default: Pipeline Succeeded
+        default: "Success"
       - name: commentfailure
         description: The text of the failure comment
-        default: Pipeline Failed
+        default: "Failed"
+      - name: dashboard-url
+        description: The URL to the pipelineruns page of the dashboard
+        default: "http://localhost:9097/"
+      # This can be deleted after pending status change
```

Comment:
I see this a lot, is this comment really needed? If it's something for an end user it may be better suited in the readme, or for developers in development.md

Reply:
# This can be deleted after pending status change issue is resolved, that being that AFAIK the pull request resource only modifies
      # status once everything is complete, so we can only modify status via the pull request resource once.  To get around this we hit
      # the github status URL to set the status into pending and use this secret to during that request.